src/Explorer/Menus/CommandBar/ConnectionStatusComponent.tsx

```diff
@@ -1,17 +1,21 @@
 import { Icon, ProgressIndicator, Stack, TooltipHost } from "@fluentui/react";
+import { ActionButton } from "@fluentui/react/lib/Button";
 import * as React from "react";
+import "../../../../less/hostedexplorer.less";
 import { ConnectionStatusType } from "../../../Common/Constants";
+import Explorer from "../../Explorer";
 import { useNotebook } from "../../Notebook/useNotebook";
 import "../CommandBar/ConnectionStatusComponent.less";
-
-export const ConnectionStatus: React.FC = (): JSX.Element => {
+interface Props {
+  container: Explorer;
+}
+export const ConnectionStatus: React.FC<Props> = ({ container }: Props): JSX.Element => {
   const [second, setSecond] = React.useState("00");
   const [minute, setMinute] = React.useState("00");
   const [isActive, setIsActive] = React.useState(false);
   const [counter, setCounter] = React.useState(0);
-  const [statusColor, setStatusColor] = React.useState("locationYellowDot");
-  const [statusColorAnimation, setStatusColorAnimation] = React.useState("ringringYellow");
-  const toolTipContent = "Hosted runtime status.";
+  const [statusColor, setStatusColor] = React.useState("status connecting is-animating");
```

Comment:
>connecting [](http://example.com/codeflow?start=63&length=10)

why default is connecting?

Reply:
This is the first status, rendered only when connecting state.[Removed].